src/Servers/Kestrel/tools/CodeGenerator/HttpProtocolFeatureCollection.cs

```diff
@@ -49,7 +49,8 @@ public static string GenerateFile()
                 "IHttpMinResponseDataRateFeature",
                 "IHttpBodyControlFeature",
                 "IHttpRequestBodyDetectionFeature",
-                "IHttpResetFeature"
+                "IHttpResetFeature",
+                "IHttpActivityFeature"
```

Comment:
Why is this a maybe feature? It should be an always feature right?

Reply:
No, the feature is only set if an activity is created. (also there really should be comments about these different sections, maybe vs. sometimes, wtf?)

Reply:
If the Activity property were settable on the feature then Kestrel could always provide a default implementation of the feature with no value.

Reply:
@Tratcher gets it

Reply:
>API approved based on the updated design. We want the feature to be nullable, not the Activity itself.

Do we need to bring this back to api review then?

Reply:
Back to API review it is.

Reply:
I assume we're making it a _known feature_ on all other servers?